src/extension.js

```diff
@@ -360,53 +354,56 @@ class ServiceIndicator extends QuickSettings.SystemIndicator {
     }
 });
 
-var serviceIndicator = null;
-var lockscreenInput = null;
-
-function init() {
-    // If installed as a user extension, this checks the permissions
-    // on certain critical files in the extension directory
-    // to ensure that they have the executable bit set,
-    // and makes them executable if not. Some packaging methods
-    // (particularly GitHub Actions artifacts) automatically remove
-    // executable bits from all contents, presumably for security.
-    Utils.ensurePermissions();
-
-    // If installed as a user extension, this will install the Desktop entry,
-    // DBus and systemd service files necessary for DBus activation and
-    // GNotifications. Since there's no uninit()/uninstall() hook for extensions
-    // and they're only used *by* GSConnect, they should be okay to leave.
-    Utils.installService();
-
-    // These modify the notification source for GSConnect's GNotifications and
-    // need to be active even when the extension is disabled (eg. lock screen).
-    // Since they *only* affect notifications from GSConnect, it should be okay
-    // to leave them applied.
-    Notification.patchGSConnectNotificationSource();
-    Notification.patchGtkNotificationDaemon();
-
-    // This watches for the service to start and exports a custom clipboard
-    // portal for use on Wayland
-    Clipboard.watchService();
-}
-
-
-function enable() {
-    serviceIndicator = new ServiceIndicator();
-    Notification.patchGtkNotificationSources();
+export let serviceIndicator = null;
+export let lockscreenInput = null;
```

Comment:
I don't think these need to be exported, you can probably just make them variables of the `GSConnectExtension` class now.

Reply:
`serviceIndicator` is being used from another class, but I've moved `lockscreenInput` in